extensions/amp-story/1.0/amp-story.js

```diff
@@ -579,6 +576,9 @@ export class AmpStory extends AMP.BaseElement {
     this.topBar_.appendChild(this.buildTopBarShare_());
 
     this.element.insertBefore(this.topBar_, this.element.firstChild);
+
+    this.onCanShowSystemLayerButtonsUpdate_(
+        !!this.storeService_.get(StateProperty.CAN_SHOW_SYSTEM_LAYER_BUTTONS));
```

Comment:
Do we need to coerce this?

Reply:
If you're talking about the ``onCanShowSystemLayerButtonsUpdate_``:

Yes, the topbar is built whenever we switch to desktop mode, which could be right away if the code is loaded from a desktop browser, or later if someone resizes their browser. This method call ensures the class is applied.

If you're talking about the ``!!`` yes, to satisfy the linter and ensure we pass a ``boolean``

Reply:
I meant the `!!` I didn't see how it could ever not be a `boolean` but you know the flow better